recipes/pangeo-forge-runner/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pangeo-forge-runner" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pangeo-forge-runner-{{ version }}.tar.gz
+  sha256: 1e977d14990a8a264fe951d2b2021a74503622a277fc29e9ef1c9ab5722297be
+
+build:
+  entry_points:
+    - pangeo-forge-runner=pangeo_forge_runner.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win or py<39 or py>=310]
```

Comment:
What is the reason to skip Windows? My guess is that we can make this noarch regardless.

Reply:
Looks like the original Windows failure was due to missing apache-beam, so yes, this can be noarch and we can ignore the Windows build failure in that case.